build.gradle

```diff
@@ -146,6 +146,7 @@ dependencies {
     
     modlauncherCompile configurations.compile
     modlauncherCompile 'net.sf.jopt-simple:jopt-simple:5.0.4'
+    modlauncherCompile 'cpw.mods:grossjava9hacks:1.1.0'
```

Comment:
This change is unnecessary. This dependency is included transitively by modlauncher.

Reply:
But for how long?

Reply:
I personally have building errors not able to find the Java9 hack classes without it. Completely wiped cache and .m2 and still happening. Not sure if my gradle version or what is causing it. But this fixes it fully.